modules/carto/src/sources/types.ts

```diff
@@ -121,6 +126,11 @@ export type TableSourceOptions = {
    * If not present, defaults to `'geom'` for generic tables, `'quadbin'` for Quadbin sources and `'h3'` for H3 sources.
    */
   spatialDataColumn?: string;
+
+  /**
+   * Relative resolution of a tile. Higher values increase density and data size. Default = 1.
```

Comment:
What about explaining it in more detail? 

When tileResoultion=`1` corresponds to a simplification where the coordinates within a tile a snapped to a 1024x1024 grid. `0.5` corresponds to a 512x512 grid, `4` 4096x4096 , etc...

Reply:
Done!